### DISCUSSION TITLE
Adding a newsletter (email form) footer

### DISCUSSION THREAD

**Discussion by pressslav** [chrismwilliams/astro-theme-cactus#247] [Q&A]
Hey Chris! Thank you for creating an awesome theme!

Do you have any idea of how to quickly add a newsletter form (email subscription) to the bottom of a page?

Some simple along these lines:

![image](https://github.com/chrismwilliams/astro-theme-cactus/assets/55917783/506a4382-b8b9-43e1-a8e5-0d45f02d2159)

**Comment by chrismwilliams** [comment]
Hi @pressslav, and thank you. I've only just seen this so apologies, but it looks like you're sorted.
